utils/hct/hctbuild.cmd

```diff
@@ -179,6 +177,22 @@ if "%1"=="-dxc-cmake-ends-include" (
   shift /1
 )
 
+if "%1"=="-dxc-cmake-path" (
```

Comment:
From the name, I might imagine this is the directory cmake is found in, but it seems to expect that cmake.exe be included. Maybe when checking below, we could check that the provided path exists?

Reply:
I would prefer it includes the whole path including cmake.exe because that is what CMAKE_COMMAND cmake variable is set to when cmake generates a project. Therefore the value of CMAKE_COMMAND can be passes directly to -dxc-cmake-path without reparsing when hctbuild is invoked by another project's CMakeLists.txt. I would also prefer to not include cmake.exe existence testing since the value of CMAKE_COMMAND uses forward slashes which works for invocation but not for mere existence testing ("if exists" batch command expects backward slashes in the path). Would you prefer a different name of the argument?

Reply:
Maybe just -dxc-cmake. I think that would be clearer, but it's a minor quibble and I won't insist.